commerce-discount-service/src/main/java/com/liferay/commerce/discount/internal/CommerceDiscountCalculationImpl.java

```diff
@@ -144,8 +141,7 @@ public CommerceDiscountValue getProductCommerceDiscountValue(
 		}
 
 		SearchContext searchContext = buildSearchContext(
-			cpInstance.getCompanyId(),
-			commerceContext.getCommerceChannelGroupId(),
+			cpInstance.getCompanyId(), cpInstance.getGroupId(),
```

Comment:
@alecsloan the pInstance.getGroupId() is not the same of getCommerceChannelGroupId, why we are changing this?

Reply:
This commit shouldn't be in this pull. Removing and force-pushing